### DISCUSSION TITLE
[Suggestion] Public Mayor Elections data

### DISCUSSION THREAD

**Discussion by MartinNemi03** [HypixelDev/PublicAPI#308] [Ideas]
I suggest adding **mayor election** data with _canditates_, _their perks_, _votes_ (total and percentage) and also _currently elected mayor_.

**Comment by shadowtime2000** [comment]
I don't think perks should be added just like pet abilities aren't added.

**Comment by MartinNemi03** [comment]
I would be for those being available, but it's mostly a minor thing.

**Comment by Myzumi** [comment]
it would help alot adding the perks into it as i think there will be new npcs and you don't need to add the text yourself

**Comment by bobbify** [comment]
It could just say whether perk1 was active or not, It doesn't need to show the whole description.
